types/react-bootstrap-table/react-bootstrap-table-tests.tsx

```diff
@@ -779,7 +779,7 @@ class MyCustomBody extends React.Component<MyCustomBodyProps> implements ModalBo
   getFieldValue() {
     const newRow: Partial<Product> = {};
     this.props.columns.forEach((column, i) => {
-      newRow[column.field] = (this.refs[column.field] as HTMLInputElement).value;
+      newRow[column.field] = (this.refs[column.field] as HTMLInputElement).value as number & string;
```

Comment:
@sandersn Is it actually possible for a value to be of type number and type string at the same time? I would have thought `number | string` to be more appropriate.

Reply:
Yep, it's definitely impossible! This cast is just to tell the compiler we know what we're doing -- specifically, because `HTMLInputElement.value : string`, we are asserting that `newRow[column.field]` can safely be assigned `string` even though the compiler conservatively guesses its type to be `string & number` (the intersection of types of all properties of `newRow`). This isn't generally true, so in real code we'd want to guard with `if (typeof newRow[column.field] === typeof 'string')`